lib/ansible/modules/cloud/vmware/vmware_guest.py

```diff
@@ -146,6 +146,13 @@
     - ' - C(type) (string): The type of CD-ROM, valid options are C(none), C(client) or C(iso). With C(none) the CD-ROM will be disconnected but present.'
     - ' - C(iso_path) (string): The datastore path to the ISO file to use, in the form of C([datastore1] path/to/file.iso). Required if type is set C(iso).'
     version_added: '2.5'
+  floppy:
+    description:
+    - A floppy configuration for the VM.
+    - 'Valid attributes are:'
+    - ' - C(type) (string): The type of floppy, valid options are C(none), C(client) or C(flp). With C(none) the floppy will be disconnected but present.'
+    - ' - C(flp_path) (string): The datastore path to the flp file to use, in the form of C([datastore1] path/to/file.flp). Required if type is set C(flp).'
```

Comment:
Add a note that while cloning from template, editing floppy setting is not allowed.